app/forms/CASForm.scala

```diff
@@ -0,0 +1,15 @@
+package forms
+
+import model.CASLookup
+import play.api.data.Forms._
+import play.api.data._
+
+object CASForm {
+  def apply(): Form[CASLookup] = Form(
+    "cas" -> mapping(
+      "number" -> nonEmptyText,
+      "lastName" -> nonEmptyText,
```

Comment:
I'm not sure that this should be required - usually we only require that _either_ the `postCode`, or failing that, the `lastName` is supplied.